cloud/cpi_cmd_runner.go

```diff
@@ -100,26 +102,19 @@ func (r *cpiCmdRunner) Run(context CmdContext, method string, apiVersion int, ar
 	if err != nil {
 		return CmdOutput{}, bosherr.WrapErrorf(err, "Marshalling external CPI command input %#v", cmdInput)
 	}
-	useIsolatedEnv := true
-	value, present := os.LookupEnv("BOSH_CPI_USE_ISOLATED_ENV")
-	if present {
-		useIsolatedEnv, err = strconv.ParseBool(value)
-		if err != nil {
-			return CmdOutput{}, bosherr.WrapErrorf(err, "Parsing $BOSH_CPI_USE_ISOLATED_ENV error, could not parse value: %v", value)
-		}
-	}
 
 	cmdPath := r.cpi.ExecutablePath()
 	cmd := boshsys.Command{
 		Name: cmdPath,
 		Env: map[string]string{
 			"BOSH_PACKAGES_DIR": r.cpi.PackagesDir,
 			"BOSH_JOBS_DIR":     r.cpi.JobsDir,
-			"PATH":              "/usr/local/bin:/usr/bin:/bin:/sbin",
+			"PATH":              os.Getenv("PATH"),
```

Comment:
Perhaps a default value of `"/usr/local/bin:/usr/bin:/bin:/sbin"` should remain here and we allow an override with something like `BOSH_CPI_COMPILE_ PATH`.